### PR TITLE
 Support 'main' as a default branch instead of 'master'

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: files/install-rpm-packages.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       - python3-flask
       - python3-ipdb  # for easy debugging
       - python3-jsonschema
-      #- python3-ogr  # temporary install 0.8.0 from PyPI
+      - python3-ogr
```
Comment (lachmanfrantisek) [inline files/install-rpm-packages.yaml]: Do you want to release new OGR to have your PR released?

**Review by lachmanfrantisek — APPROVED** [review]
Looks good!

And also, thanks for adding pre-commit. (I can enable the pre-commit.ci if you want.)

File: files/install-rpm-packages.yaml (hunk 1 above)
Comment (jpopelka) [reply files/install-rpm-packages.yaml]: not needed, thanks

**Comment by lioramilbaum** [comment]
@jpopelka Is there a Jira issue which tracks this effort? If not, lets create one a link it to this PR. WDYT?

**Comment by jpopelka** [comment]
We've never tracked release-bot in Jira. There's been almost no activity in this project for almost a year and I don't expect it to change so I'm not considering linking this or any other issue here with Jira.

**Pull request merged**